lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1603,6 +1637,48 @@ definitions:
         $ref: '#/definitions/network_epc_configs'
       feg_network_id:
        $ref: '#/definitions/feg_network_id'
+      ngc:
+        $ref: '#/definitions/network_ngc_configs'
+
+  suci_profile:
+    description: SUCI profile for a network
```

Comment:
Could you include a link to a specification as well?

Reply:
done